stdlib/tempfile.pyi

```diff
@@ -253,11 +253,11 @@ class _TemporaryFileWrapper(IO[AnyStr]):
     def truncate(self, size: int | None = ...) -> int: ...
     def writable(self) -> bool: ...
     @overload
-    def write(self: _TemporaryFileWrapper[str], s: str) -> int: ...
+    def write(self: _TemporaryFileWrapper[str], /, s: str) -> int: ...
```

Comment:
Why did you only mark `self` as pos-only?

Reply:
This was not my intention!

Reply:
@sobolevn I hope to have addressed this.